components/Navbar.js

```diff
@@ -1,109 +1,90 @@
-// import Image from 'next/image';
 import Link from 'next/link';
-import React from 'react';
-// import gmData from '../../config/gmData';
+import { useRouter } from 'next/router';
+import React, { useState, useEffect } from 'react';
 
-//to allow GM page to be visible on the navbar and accessible
-//uncomment the above import line "import gmData from '../../config/gmData';" to get quarter from the gmData page
-//uncomment the two lines saying  "/* <Link href="/gm"><li>{gmData.date.quarter} gm</li></Link> */"
-
-function MobileNavItem() {
-	return (
-		<li id="committee-mobile-nav-item">
-			<ul className="committee-mobile-nav">
-				<li><Link href={'/committees#studio'}><a>Studio</a></Link></li>
-				<li><Link href={'/committees#icpc'}><a>ICPC</a></Link></li>
-				<li><Link href={'/committees#design'}><a>Design</a></Link></li>
-				<li><Link href={'/committees#cyber'}><a>Cyber</a></Link></li>
-				<li><Link href={'/committees#teachla'}><a>Teach LA</a></Link></li>
-				<li><Link href={'/committees#w'}><a>W</a></Link></li>
-				<li><Link href={'/committees#ai'}><a>AI</a></Link></li>
-				<li><Link href={'/committees#hack'}><a>Hack</a></Link></li>
-			</ul>
-		</li>
-	);
-}
+export default function Navbar() {
+  // set states
+  const [menuActive, setMenuActive] = useState('');
```

Comment:
Since this is functionally a boolean (it's either `"active"` or `""`), could we make this itself a boolean state variable, and then toggle the CSS class based off of the value of the boolean? I'm a bit iffy on "stringly-typed" variables, and would like to avoid them if possible!

i.e.

```js
const [isMenuActive, setIsMenuActive] = useState(false);
// ...
const toggleMenu = () => {
    setIsMenuActive(!isMenuActive);
};
// ...
const menuActiveClass = isMenuActive ? 'active' : '';